src/components/grid/index.js

```diff
@@ -23,7 +24,7 @@ const styles = StyleSheet.create({
   },
   vertical: {
     flexDirection: 'column',
-    alignItems: 'flex-start',
+    justifyContent: 'flex-start',
```

Comment:
Is there specific case for which this is change is required?
As far as I remember, it used to be like that, but we changed it before releasing 1.0, because:
1. `justifyContent: 'flex-start'` is [the default](https://facebook.github.io/react-native/docs/layout-props#justifycontent) so there is no need to override it
2. `alignItems: 'stretch'` is [the default](https://facebook.github.io/react-native/docs/layout-props#alignitems) so we need to override it to prevent elements without specific size from stretching by default (which can still be achieved with `<Block size="stretch">`)

Reply:
I don't remember why off the top of my head, I think it had something to do with react-native-web. I'll try to do some tests before the end of next week and will let you know!